package.json

```diff
@@ -65,7 +65,6 @@
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31407&commitId=9e4e54e1f87e5c2df2327ce64796baf6179fbade&env=prod&clientType=FLUENTUI)


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>